test/e2e/auth/certificates.go

```diff
@@ -323,10 +333,14 @@ var _ = SIGDescribe("Certificates API [Privileged:ClusterAdmin]", func() {
 		for sawAnnotations := false; !sawAnnotations; {
 			select {
 			case evt, ok := <-csrWatch.ResultChan():
-				framework.ExpectEqual(ok, true, "watch channel should not close")
+				if !ok {
+					framework.Fail("watch channel should not close")
```

Comment:
Failf?

Reply:
The parameter is a static string, not a printf-style format string, so Fail is more suitable here.

Reply:
oh I see. I didn't notice above conversations. Thanks!